packages/theme-docs/src/store/index.js

```diff
@@ -86,6 +90,28 @@ export const actions = {
 
     commit('SET_RELEASES', releases)
   },
+  async fetchDefaultBranch ({ commit, state }) {
+    if (!state.settings.github || state.settings.defaultBranch) {
+      return
+    }
+
+    const options = {}
+    if (process.env.GITHUB_TOKEN) {
+      options.headers = { Authorization: `token ${process.env.GITHUB_TOKEN}` }
+    }
+    let defaultBranch
+    try {
+      const data = await fetch(`https://api.github.com/repos/${state.settings.github}`, options).then((res) => {
+        if (!res.ok) {
+          throw new Error(res.statusText)
+        }
+        return res
+      }).then(res => res.json())
+      defaultBranch = data.default_branch
+    } catch (e) {}
+
+    commit('SET_DEFAULT_BRANCH', defaultBranch || 'dev')
```

Comment:
@danielroe Maybe this should be set to `master` by default?

Reply:
That would be less opinionated, although at some point that will switch to main, and so we could just start with that?

https://github.com/github/renaming

Reply:
let me know if you prefer `master` and I can adjust PR

Reply:
`main` is fine :)